handsontable/test/e2e/Stylesheets.spec.js

```diff
@@ -0,0 +1,42 @@
+describe('Stylesheets', () => {
+  describe('CSS charset declarations', () => {
+    it('should have exactly one @charset declaration at the beginning of each CSS file', () => {
+      if (!['classic', undefined].includes(spec().loadedTheme)) {
+        pending('Test is only valid for the classic theme.');
+
+        return;
+      }
+
+      const hotCss = [
```

Comment:
What about that?
```suggestion
    it.forTheme('classic')('should have exactly one @charset declaration at the beginning of each CSS file', () => {
      const hotCss = [
```

Reply:
I was considering this, but figured it would disrupt the balance of all `forTheme` tests being run equally between themes. Then again, it should not harm anything, I'll change it.